### DISCUSSION TITLE
求助，怎么在header这里修改，只显示第一个作者

### DISCUSSION THREAD

**Discussion by wsm6636** [windingwind/zotero-better-notes#1052] [Q&A] [answered]
<img width="962" alt="Snipaste_2024-07-24_13-47-16" src="https://github.com/user-attachments/assets/2c694b0e-8187-4841-9387-4eeafef46281">
怎么不显示后面那个等

**Comment by windingwind** [comment, marked as answer]
```js
let firstCreator = noteItem.parentItem?.getCreator(0);
header.FirstAuthor = `${firstCreator?.firstName} ${firstCreator?.lastName}`;
```

**Reply by wsm6636** [reply]
> ```js
> let firstCreator = noteItem.parentItem?.getCreator(0);
> header.FirstAuthor = `${firstCreator?.firstName} ${firstCreator.?lastName}`;
> ```
谢谢！！有一个小小的笔误
> ```js
> header.FirstAuthor = `${firstCreator?.firstName} ${firstCreator?.lastName}`;
> ```

**Reply by windingwind** [reply]
谢谢！
